App/components/Image/image.tsx

```diff
@@ -1,23 +1,34 @@
 import React, { ReactNode } from 'react'
+import { ImageSourcePropType } from 'react-native'
 import * as Styled from './image.styled'
 
-type Props = {
+export type Props = {
   children: ReactNode
   image: string
   width: number
   height: number
-  border?: string
-  borderRadius?: number
+  border?: string | undefined
```

Comment:
Also: please take a look at how we use theme props in SOMMELIER_1: 
```javascript
border?: keyof typeof border;
```
And it will do a thing so it will be ok to use the border theme in a styled components.

Reply:
Kept here but changed text to select in story, bc it's actually string